app/models/importers/organisation.rb

```diff
@@ -11,19 +12,36 @@ def initialize(slug, batch: 10, start: 0)
   def run
     organisation = ::Organisation.create!(slug: slug)
     loop do
-      response = HTTParty.get "https://www.gov.uk/api/search.json?filter_organisations=#{slug}&count=#{batch}&fields=content_id&start=#{start}"
-      results = JSON.parse(response.body).fetch('results')
-
-      raise "No result for slug" if results.empty? && organisation.content_items.empty?
-
-      results.each do |result|
-        attributes = result.slice('content_id')
+      result = search_content_items_for_organisation
```

Comment:
I thought it wasn't very Rubyist to use verbs in method names? Should this be `content_items_for_organisation`?